tidy3d/updater.py

```diff
@@ -174,6 +174,27 @@ def new_update_function(sim_dict: dict) -> dict:
     return decorator
 
 
+@updates_from_version("1.4")
+def update_1_4(sim_dict: dict) -> dict:
+    """Updates version 1.3 to 1.4."""
+
+    def fix_polyslab_dict(geo_dict):
+        """Fix a single PolySlab dictionary."""
+        geo_dict.pop("length", None)
+        geo_dict.pop("center", None)
+
+    for structure in sim_dict["structures"]:
+        geo_dict = structure["geometry"]
+        if geo_dict["type"] == "PolySlab":
+            fix_polyslab_dict(geo_dict)
+        elif geo_dict["type"] == "GeometryGroup":
+            for sub_geo in geo_dict["geometries"]:
```

Comment:
need another `geo_dict.pop("center", None)` here because in old jsons `GeometryGroup` also has a center.